### DISCUSSION TITLE
How do I delete 2FA?

### DISCUSSION THREAD

**Discussion by MatyUnai** [community/community#116821] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I need to remove the verification code when logging in from another device since it does not allow me to do a migration.

**Comment by MatyUnai** [comment]
![screen1](https://github.com/community/community/assets/165736341/c6524fd9-1c2f-4a5a-88f6-6a8f9ef9d282)

**Comment by gil-son** [comment]
Hello, 

First of all, 2FA is a requisite to use Github. Is possible you change to a new Authenticator app

1. Click on the button Add in the Authenticator app
2. Type your password or the code from your Authenticator app
3. Now you can add a new verification from another Authenticator app

If I supported you, consider to mark my answer. God bless you!
